rpc.go

```diff
@@ -194,17 +195,24 @@ func (t *Topic) Publish(
 	}
 	t.lk.Lock()
```

Comment:
We can wrap this with `if !args.ignoreResponse || args.republish {` to avoid adding the entry at all, but no big deal.

Reply:
oh yeah, cool. pushed that changed